qa/tasks/cephadm.py

```diff
@@ -1161,8 +1161,16 @@ def task(ctx, config):
     ref = None
     if not ctx.ceph[cluster_name].image:
         sha1 = config.get('sha1')
+        # right now we only have support for centos 8 images
+        os = config.get('os_type')
+        os_ver = config.get('os_version')
+        if 'flavor' not in config:
+            config['flavor'] = 'default'
+        flavor = config.get('flavor')
+
         if sha1:
-            ctx.ceph[cluster_name].image = container_image_name + ':' + sha1
+            #eg: quay.ceph.io/ceph-ci/ceph:8f1620b3c9322a518ba8b8c6573435d81b9c0756:centos-8:default
```

Comment:
this should look like so:

```suggestion
            #eg: quay.ceph.io/ceph-ci/ceph:8f1620b3c9322a518ba8b8c6573435d81b9c0756-centos-8-default
```

IMO we shouldn't use colons in the image tag itself.